docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -68,6 +68,23 @@
 |==============================================
 // end::create-exception-container-request-body[]
 
+// tag::create-host-isolation-exception-container-request-body[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`description` |String |Describes the {endpoint-artifact-name} container. |Yes
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
```

Comment:
```suggestion
|`list_id` |String |Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
```

Reply:
Weirdly I couldn't see where this extra A appears in the text, but it seems extra 🤷

Reply:
Yeah, this definitely looks like a minor typo. Good catch, thanks!